container/root/usr/local/bin/archstring

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Script to detect processor architecture and output one of the two input parameters
+# Example: when a download path requires an arch-specific package name
+#
+# --x64 {value}: output if an x86_64 architecture is detected
+# --arm64 {value}: output if an arm64 or aarch64 architecture is detected
+function usage() {
+  echo "usage:a $(basename $0) --x64 value --arm64 value"
```

Comment:
@bryanlatten : is that `a` in `usage:a` a typo? Or should that be there?
`echo "usage: $(basename $0)`

Reply:
seems like a strange addition. fixed.